src/AppBundle/Entity/EventRegistration.php

```diff
@@ -104,6 +105,15 @@ public function isNewsletterSubscriber(): bool
         return $this->newsletterSubscriber;
     }
 
+    public function isEventFinished(): bool
+    {
+        if (!$this->event instanceof Event) {
```

Comment:
Useless the `Event` object is required by the constructor, am I missing something?